src/SFML/System/Utils.cpp

```diff
@@ -44,8 +44,10 @@ std::string toLower(std::string str)
 std::string formatDebugPathInfo(const std::filesystem::path& path)
 {
     std::ostringstream oss;
-    oss << "    Provided path: " << path << '\n' //
-        << "    Absolute path: " << std::filesystem::absolute(path);
+    // convert to UTF-8 to handle non-ascii/non-latin1 filenames on windows
+    // cast to work in C++20 where u8string is char8_t which can't be printed to char stream
```

Comment:
```suggestion
    // cast is required to work in C++20 where u8string is char8_t which can't be printed to char stream
```

Reply:
This comment change was done and merged in separate PR https://github.com/SFML/SFML/pull/3407 and this commit is now gone from this PR (we're down to 2).